metadata/src/show.rs

```diff
@@ -63,7 +63,10 @@ impl TryFrom<&<Self as Metadata>::Message> for Show {
             media_type: show.media_type(),
             consumption_order: show.consumption_order(),
             availability: show.availability.as_slice().try_into()?,
-            trailer_uri: SpotifyId::from_uri(show.trailer_uri())?,
+            trailer_uri: Some(show.trailer_uri())
```

Comment:
The field `show.trailer_uri` (not the method) is of type `Option`. Could you check if the field is `None` in the faulty case? If so its probably better to handle the `Option` instead of the empty string. Usually the generated protobuf methods return the default value if the underlying `Option` is None.

> calling `as_deref` on the Option might be of help as protobuf fields are usually `String` and `SpotifyId::from_uri` only accepts `&str`

Reply:
Oh wait nvm, just saw your attached dump of the obj. And as expected its `None` and not an empty string. Could you adjust the code to handle the `Option` instead of the string then?

Reply:
That was actually my first attempt (using `has_trailer_uri`), but in this case, it actually was `Some("")`, so that didn't help.

Reply:
What a bummer. You could technically still access the field directly instead of wrapping the function output. But that's just nitpicking, so just resolve this comment when you want to keep it as is.

Reply:
I changed it to the direct access, that indeed seems like the cleaner way.